ui/shared/RevisionList.jsx

```diff
@@ -4,9 +4,44 @@ import { Col, Row } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
 
+import { getData } from '../helpers/http';
+import { bugzillaBugsApi } from '../helpers/url';
+
 import { Revision } from './Revision';
 
 export class RevisionList extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      revisionComments: {},
+    };
+  }
+
+  componentDidMount() {
+    const { revisions } = this.props;
+
+    revisions.forEach((revision, i) => {
+      this.getRevisionComment(revision.comments, i);
+    });
+  }
+
+  getRevisionComment = async (comments, i) => {
+    const comment = comments.split('\n')[0];
+    const bugMatches = comment.match(/-- ([0-9]+)|bug.([0-9]+)/gi);
+    var bugNumbers = '';
+    var bugData = {};
+    bugMatches.forEach((bugMatch) => {
```

Comment:
Instead of declaring the variable of `bugNumbers` and using `forEach` to populate it, the preferred technique is to use `const bugNumbers = ` and then use `bugMatches.map`.